cdap-app-fabric/src/test/java/co/cask/cdap/internal/guice/AppFabricTestModule.java

```diff
@@ -131,6 +131,7 @@ protected void configure() {
     // we want to use RemotePrivilegesFetcher in this module, since app fabric service is started
     install(new AuthorizationEnforcementModule().getStandaloneModules());
     install(new SecureStoreModules().getInMemoryModules());
+    install(new PreviewServerModule());
```

Comment:
This is required since in `PreviewHttpHandlerTest` we get the instance of `PreviewHttpHandler` using injector and `PreviewHttpHandlerTest` extends from `AppFabricTestBase`. 
https://github.com/caskdata/cdap/blob/develop/cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/PreviewHttpHandlerTest.java#L32